### ISSUE TITLE
Mismatch with documentation

### ISSUE THREAD

**Issue by adhalanay** [terminusdb/terminusdb-dashboard#311]
I don't know if this the right place to file it so if it's not pleas feel free to close it. In the documentation, more precisely [here](https://terminusdb.com/docs/use-the-model-builder-ui/) appears that one is able to visually add classes and links and also add properties for them. In the current dashboard both locally installed and on TerminusCMS, this does not seem to be possible.

**Comment by alexander-mart** [comment]
@adhalanay Yes, it is possible! You can feel free to ask any questions if you want.

@hoijnet @GavinMendelGleason we can close it now
